packages/data-store/src/migrations/2.simplifyRelations.ts

```diff
@@ -8,22 +8,120 @@ import Debug from 'debug'
  */
 export class SimplifyRelations1447159020002 implements MigrationInterface {
   async up(queryRunner: QueryRunner): Promise<void> {
+    function getTable(givenName: string): Table {
+      const entityMetadatas = queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName);
+      return Table.create(entityMetadatas!, queryRunner.connection.driver);
+    }
     function getTableName(givenName: string): string {
       return (
-        queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName)?.tableName ||
-        givenName
+          queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName)?.tableName ||
+          givenName
       )
     }
-    await queryRunner.changeColumn(
-      getTableName('key'),
-      'identifierDid',
-      new TableColumn({ name: 'identifierDid', type: 'varchar', isNullable: true }),
+
+    await queryRunner.startTransaction();
```

Comment:
Do not manually start transactions in migration files. Typorm already provides transaction configuration support for migrations